alica_viewer/CMakeLists.txt

```diff
@@ -88,4 +88,8 @@ install(DIRECTORY include/${PROJECT_NAME}/
 
 install(DIRECTORY launch/
         DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
+
+install(FILES ${UIS_HEADERS}
+        DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
```

Comment:
will only recommend global_include_destination if package_include aren't working.

Reply:
I think package include will not work here unless we also change `#include <ui_AlicaWidget.h>` in alica_client/include/alica/AlicaWidget.h